### ISSUE TITLE
Correct way of representing long data in a data.tree

### ISSUE THREAD

**Issue by hannes101** [gluc/data.tree#127]
Hi,
thanks for that nice package and your work. I am struggling to import a data.table into the data.tree format.
I described the problem on SO, but I doubt that there are a lot of people knowing data.tree.
https://stackoverflow.com/questions/53535918/importing-long-data-into-data-tree-structure
I now came up with the idea that I have to add the Trade Status to the relationship path and I think that's correct, or?
```
library(data.table)
dt.tree.input <- structure(list(msg_seq_nb = c("0005747", "0005747", "0005747", "0005747")
               , Initial.Trade.Status = c("T", "T", "C", "C")
               , Initial.Trd.Rpt.Dt = structure(c(15987, 15987, 15987, 15987), class = "Date")
               , Initial.Trd.Rpt.Tm = c(34838, 34838, 34853, 34853)
               , J2.Msg.Nb = c("0005765", "0005765", "0005765", "0005765")
               , J2.Trade.Status = c("R", "C", "R", "C")
               , J2.Trd.Rpt.Tm = c(34853, 34863, 34853, 34863)
               , pathString = c("Trade/0005747/0005765", "Trade/0005747/0005765", "Trade/0005747/0005765", "Trade/0005747/0005765"))
          , row.names = c(NA, -4L), class = c("data.table", "data.frame"))
```
Basically I am adding the Trade Status to the pathString and thus create another level of leaves.
```
dt.tree.input[, pathString :=  paste("Trade", msg_seq_nb
                                       , J2.Msg.Nb, J2.Trade.Status, sep  = "/", na.rm = TRUE) ]
```
This then results in the following tree structure, is this correct?

```
test.Node <- as.Node(dt.tree.input)
print(test.Node, "Initial.Trade.Status", "J2.Trade.Status")
        levelName Initial.Trade.Status J2.Trade.Status
1 Trade                                               
2  °--0005747                                         
3      °--0005765                                     
4          ¦--R                      C               R
5          °--C                      C               C
```

**Comment by gluc** [comment]
Thx for this, and sorry for the late reply. Looks like this works according to your requirements? There might be better ways, depending on how big your data is and what you want to achieve.
